src/utils/self_test/self_test.c

```diff
@@ -11,10 +11,21 @@
 #include <getopt.h>
 #include <string.h>
 #include <math.h>
+#include <ctype.h>
 
 #include "tests_common.h"
+#include "configini.h"
 #include "daos_errno.h"
 
+/* Section names */
+#define DEFAULT_SCALE_NAME			"scale"
```

Comment:
i would prefer scale to be renamed threshold

Reply:
Name has been changed.